### ISSUE TITLE
Consistent wording on cards with no revenue/production/disbursements

### ISSUE THREAD

**Issue by jennmalcolm** [ONRR/nrrd#603]
A couple concerns on the production cards:

- The writing should span the width of the card.
- Oil (or other commodity) should not be capitalized, and we don't need the unit here.
- The wording makes it sound like oil was produced in 2003. I would change it to be more consistent with the revenue statement. In this case, it could say, "This location has no federal oil production." We could also be consistent with what disbursements is saying now and say "no production" instead.

![image.png](https://images.zenhubusercontent.com/5c59c5bee180620812ea29eb/705c409d-7567-4fca-a64f-0325edc7d14e)

For disbursements and revenue, we should be consistent either with a full sentence or not. ![image.png](https://images.zenhubusercontent.com/5c59c5bee180620812ea29eb/658b156d-9f1a-4afb-8dde-68e88d1cd614) ![image.png](https://images.zenhubusercontent.com/5c59c5bee180620812ea29eb/3215c692-b754-40a8-b937-d1b117b79dc4)

**Comment by jennmalcolm** [comment]
How about this wording?

(Location) had no federal (commodity or commodities if selected) (revenues/disbursements/production) during (FY/CY) (year).

**Comment by mcharg** [comment]
We should still show the sparkline if there was no data for the current year, but there is data for other years.  I think what we have for Tennessee works for that scenario, unless we want to replace the 0 with a dash.
![image](https://user-images.githubusercontent.com/38465926/87308840-d1880d80-c4e9-11ea-8015-3245435465bf.png)

The message should be for when there is data in no years.

Something like:
(Location) had no federal (commodity or commodities if selected) (revenues/disbursements/production) during the years for which we have data.

**Comment by mojobnichols** [comment]
Added Jerome since he fixed Production and Disbursements

**Comment by mcharg** [comment]
Production is fixed in @jlouis-onrr's PR, but not disbursements.
